### PR TITLE
replace  browserify with webpack for bulding dist/

### PR DESCRIPTION
**Pull request by hipertracker**
I did not add new dist/*js files. They have to be regenerated and added to repo.

### CODE REVIEW

**Comment by hipertracker** [comment]
Well, my change definitely works. But for Travis CI it needs something more... Weird.

**Comment by jdlehman** [comment]
Thanks for picking this up, my branch fell pretty far behind haha.

**Comment by goatslacker** [comment]
Can we place these in a config directory perhaps? Or no?

File: package.json [hunk 1]
```diff
@@ -38,14 +41,14 @@
   },
   "authors": [
     "Josh Perez <josh@goatslacker.com>",
-    "Jonathan Lehman <jonathan.lehman91@gmail.com>"
+    "Jonathan Lehman <jonathan.lehman91@gmail.com>",
+    "Jaroslaw Zabiello <hipertracker@gmail.com>"
```
Comment (goatslacker) [inline package.json]: This space is for the project maintainers not all the [contributors](https://github.com/goatslacker/alt/graphs/contributors) although I'm not opposed to adding a link to that page or mentioning it somewhere else :)

File: package.json [hunk 2]
```diff
@@ -9,8 +9,9 @@
...
   "devDependencies": {
     "alt-search-docs": "1.0.6",
-    "babel": "^5.3.1",
-    "babel-core": "^5.3.1",
+    "babel": "^5.6.6",
+    "babel-core": "^5.6.6",
+    "babel-loader": "^5.1.4",
     "babelify": "^6.1.0",
     "browserify": "^10.1.3",
     "browserify-shim": "^3.8.6",
```
Comment (goatslacker) [inline package.json]: we can rm this now right?

File: package.json [hunk 3]
```diff
@@ -9,8 +9,9 @@
   },
   "devDependencies": {
     "alt-search-docs": "1.0.6",
-    "babel": "^5.3.1",
-    "babel-core": "^5.3.1",
+    "babel": "^5.6.6",
```
Comment (goatslacker) [inline package.json]: Let me peek through changelog before we do this.

File: package.json (hunk 2 above)
Comment (jdlehman) [inline package.json]: I think babel-core is still needed as it is a peer dependency to babel-loader, but I am not 100%

Comment (goatslacker) [inline package.json]: browserify-shim i mean

File: package.json (hunk 1 above)
Comment (hipertracker) [inline package.json]: I didn't know. :) You can delete it.

**Pull request merged**